jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java

```diff
@@ -81,9 +80,10 @@ public void testFromPluginConfigurationPull() {
     public void testFromPluginConfigurationFailed() {
         FromConfigRegistryAuthHandler handler = new FromConfigRegistryAuthHandler(
             RegistryAuthConfig.builder().putDefaultConfig(RegistryAuth.USERNAME, "admin").build(), log);
-
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage(containsString("password"));
+        
+        IllegalArgumentException exception= assertThrows(IllegalArgumentException.class, () -> handler.create(RegistryAuthConfig.Kind.PUSH, null, null, s -> s));             
```

Comment:
You've put `handler.create()` call in `assertThrows` now but you forgot to remove previous `handler.create` call(on line 87).

This is causing test to fail. Please remove it.